src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,116 @@
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (
+    artists: FeaturedArtists_collection["artworksConnection"]["merchandisableArtists"]
+  ) => {
+    return artists.map((artist, index) => {
+      const hasArtistMetaData = artist.nationality && artist.birthday
+      return (
+        <Box width="100%" key={index} pb={20}>
+          <TouchableWithoutFeedback onPress={() => this.handleTap(this, `/artist/${artist.slug}`)}>
+            <EntityHeader
+              imageUrl={artist.image.resized.url}
+              name={artist.name}
+              meta={hasArtistMetaData ? `${artist.nationality}, b. ${artist.birthday}` : undefined}
+              href={`/artist/${artist.slug}`}
+              FollowButton={
+                <Button
+                  variant={artist.isFollowed ? "primaryBlack" : "secondaryOutline"}
+                  size="small"
+                  longestText="Following"
+                >
+                  {artist.isFollowed ? "Following" : "Follow"}
+                </Button>
+              }
+            />
+          </TouchableWithoutFeedback>
+        </Box>
+      )
+    })
+  }
+
+  render() {
+    const artists = get(this.props, p => p.collection.artworksConnection.merchandisableArtists)
+    if (!artists) {
+      return null
+    }
+
+    const hasMultipleArtists = artists.length > 1
+
+    const artistCount = 3
+    const remainingCount = artists.length - artistCount
+    const truncatedArtists = this.getFeaturedArtistEntityCollection(artists).slice(0, artistCount)
+    const headlineLabel = "Featured Artist" + (hasMultipleArtists ? "s" : "")
+
+    return (
+      <Box pb={1}>
+        <Sans size="2" weight="medium" pb={15}>
+          {headlineLabel}
+        </Sans>
+        <Flex flexWrap="wrap">
+          {this.state.showMore || artists.length <= artistCount ? (
+            this.getFeaturedArtistEntityCollection(artists)
+          ) : (
+            <>
+              {truncatedArtists}
+              <TouchableHighlight
+                onPress={() => {
+                  this.setState({ showMore: true })
+                }}
+              >
+                <EntityHeader initials={`+ ${remainingCount}`} name="View more" />
+              </TouchableHighlight>
+            </>
+          )}
+        </Flex>
+      </Box>
+    )
+  }
+}
+
+export const CollectionFeaturedArtistsContainer = createFragmentContainer(FeaturedArtists, {
+  collection: graphql`
+    fragment FeaturedArtists_collection on MarketingCollection {
+      # TODO: size:9 is not actually limiting to 9 items. We need to figure out
+      #  why the back-end is not respecting that argument.
+      artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 9, sort: "-decayed_merch") {
+        merchandisableArtists {
+          slug
+          internalID
+          name
+          image {
+            resized(width: 100) {
```

Comment:
@ashfurrow @kierangillen We noticed while writing tests that we were using the full `imageUrl`. Having been bitten fairly recently by unresized images on web, we thought it would be appropriate to resize here also. Is there any reason not to resize, and is the width we chose reasonable for an entity header image on mobile?

Reply:
bump @ashfurrow.

Reply:
Thanks for the bump! I'm curious about the undersized images issue, but let me answer your questions:

> Is there any reason not to resize?

None that I can see. The `resized` resolver isn't used commonly in Emission yet, but I think that's because most of Emission predates this MP resolver.

> is the width we chose reasonable for an entity header image on mobile?

I don't think it's sufficient. The narrowest iPhone is 320 points wide (640 actual pixels). I would say 500 would be a better value here. 

_Ideally_, we would pass in the actual screen width by importing `Dimensions` from the `react-native` module and use `Dimensions.get("window").width`. This could be passed in to the fragment with an argument with `@arguments` wherever `FeaturedArtists_collection` is used. This way, the image would match exactly the screen size. This would also account for devices like iPad that have really high screen widths. Let me know if I can clarify/help/pair.